solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/listProjectsByEtc.test.ts

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Nit: tests exist under `integration-tests/tests/isolated/projectEnvTypeConfigs/listProjects.test.ts`. Look to consolidate these.

Reply:
Ah I did not know there's an existing one. Will consolidate.